spec/subscribers/super_good/solidus_taxjar/spree/reporting_subscriber_spec.rb

```diff
@@ -26,11 +26,7 @@
     end
 
     context "reporting is disabled" do
-      before do
-        allow(::SuperGood::SolidusTaxjar)
-          .to receive(:reporting_enabled)
-          .and_return(false)
-      end
+      let(:reporting_enabled) { false }
```

Comment:
I don't think this line is doing anything right now because we only reference this in the `before` for the parallel context above. I believe we're just getting the default value of `false` here when a new configuration is created the first time it's accessed.